python/tvm/relay/op/nn/nn.py

```diff
@@ -1976,6 +1976,74 @@ def contrib_conv2d_winograd_without_weight_transform(data,
         kernel_layout, out_layout, out_dtype)
 
 
+def contrib_conv2d_gemm_without_weight_transform(data,
+                                                 weight,
+                                                 strides=(1, 1),
+                                                 padding=(0, 0),
+                                                 dilation=(1, 1),
+                                                 groups=1,
+                                                 channels=None,
+                                                 kernel_size=None,
+                                                 data_layout="NCHW",
+                                                 kernel_layout="OIHW",
+                                                 out_layout="",
+                                                 out_dtype=""):
+    r"""2D convolution with gemm algorithm.
```

Comment:
All the other help texts are represented as a raw strings (i.e., starting with an "r"). I think this is because raw strings leave  backslashes in the string, and the string can be parsed later by help formatter tools. In this particular case, I don't need a raw string, since I don't have any backslash in the help text, so if you wish I can remove it

Reply:
Thanks for explanation. No need to remove.